src/main/kotlin/NotesApp.kt

```diff
@@ -0,0 +1,137 @@
+class NotesApp {
```

Comment:
Это можно сделать объектом, чтобы не создавать инстанс класса. В твоей программе всегда инстанс только 1, поэтому:
```suggestion
object NotesApp {
```